Nautilus/Documentation/guides/assetbundles.md

```diff
@@ -0,0 +1,282 @@
+```csharp
+myCustomPrefab.SetGameObject(myAssetBundleGO);
+```
+Instead of using a prefab template.  
+<br>
+For more complicated prefabs (such as buildables), you can create your own method that loads the prefab from the asset bundle and applies extensive modifications to it.  
+You can *also* apply changes to the prefab directly.
+
+E.g., ``myAssetBundleGO.AddComponent<WorldForces>();``
+
+Here is one example of how to use an Asset Bundle GameObject in a custom prefab:
+```csharp
+using Nautilus.Assets;
+using Nautilus.Assets.Gadgets;
+using Nautilus.Crafting;
+using Nautilus.Utility;
+using UnityEngine;
+using Ingredient = CraftData.Ingredient;
+
+namespace ExamplePrefab
+{
+    internal static class MyCoolPrefab
+    {
+        public static TechType techType { get; private set; }
+
+        public static void Patch()
+        {
+            PrefabInfo prefabInfo = PrefabInfo.WithTechType("myCoolPrefab", "My Cool Prefab", "Pretty cool, right!")
+                .WithIcon(SpriteManager.Get(TechType.Titanium));
+            //Just using the Titanium sprite as a placeholder
+
+            //Cache the tech type for use in other places
+            techType = prefabInfo.TechType;
+
+            var prefab = new CustomPrefab(prefabInfo);
+
+            //Create the recipe
+            RecipeData recipe = new RecipeData
+            {
+                craftAmount = 1,
+                Ingredients =
+                {
+                    new Ingredient(TechType.Titanium, 2),
+                    new Ingredient(TechType.CopperWire, 2),
+                },
+            };
+
+            //Set the prefab GamrObject to the result of the GetAssetBundlePrefab method
+            prefab.SetGameObject(GetAssetBundlePrefab());
+
+            //Using the Seaglide as a placeholder unlock
+            prefab.SetUnlock(TechType.Seaglide);
+            
+            //Set the recipe
+            prefab.SetRecipe(recipe)
+                .WithCraftingTime(6f);
+
+            //Add the prefab to the Miscellaneous tab of the blueprints in the PDA
+            prefab.SetPdaGroupCategory(TechGroup.Miscellaneous, TechCategory.Misc);
+
+            prefab.Register();
+        }
+
+        private static GameObject GetAssetBundlePrefab()
+        {
+            GameObject myCoolPrefab = assetBundle.LoadAsset<GameObject>("myCoolPrefab");
+
+            //The classID is the same as the one we put into the PrefabInfo.WithTechType up above
+            //The LargeWorldEntity.CellLevel determines how far away the object will be loaded from the player
+            PrefabUtils.AddBasicComponents(myCoolPrefab, "myCoolPrefab", techType, LargeWorldEntity.CellLevel.Medium);
+
+            //Makes the GameObject have the correct shaders
+            //You can use the optional inputs here to change the look of your object
+            MaterialUtils.ApplySNShaders(myCoolPrefab);
+
+            //Return the GameObject with all the components added
+            return myCoolPrefab;
```

Comment:
Two suggestions here:
1. When creating the custom prefab, cache the PrefabInfo instead of JUST the TechType.
2. Spaces after "//"